documentation/modules/configuring/ref-list-of-kafka-cluster-resources.adoc

```diff
@@ -33,20 +33,18 @@ The following resources are created by the Cluster Operator in the Kubernetes cl
 `_cluster-name_-kafka`:: StatefulSet which is in charge of managing the Kafka broker pods.
 `_cluster-name_-kafka-_idx_`:: Pods created by the Kafka StatefulSet.
 `_cluster-name_-kafka-brokers`:: Service needed to have DNS resolve the Kafka broker pods IP addresses directly.
-`_cluster-name_-kafka-bootstrap`:: Service can be used as bootstrap servers for Kafka clients.
-`_cluster-name_-kafka-external-bootstrap`:: Bootstrap service for clients connecting from outside the Kubernetes cluster. This resource is created only when an external listener is enabled.
-`_cluster-name_-kafka-_pod-id_`:: Service used to route traffic from outside the Kubernetes cluster to individual pods. This resource is created only when an external listener is enabled.
-`_cluster-name_-kafka-external-bootstrap`:: Bootstrap route for clients connecting from outside the Kubernetes cluster. This resource is created only when an external listener is enabled and set to type `route`.
-`_cluster-name_-kafka-_pod-id_`:: Route for traffic from outside the Kubernetes cluster to individual pods. This resource is created only when an external listener is enabled and set to type `route`.
```

Comment:
This should not be deleted probably as it is about Routes. The situation with Routes is the same as with the services. These names would be used for listener with type `route`, name `external` and port `9094`. For other listeners of type `route` but with different name and/or port the route will have the listener name in its name.